lib/util/http/github.ts

```diff
@@ -394,14 +394,13 @@ export class GithubHttp extends Http<GithubHttpOptions> {
       body,
       headers: { accept: options?.acceptHeader },
     };
-
+    if (options.token) {
+      opts.token = options.token;
+    }
     logger.trace(`Performing Github GraphQL request`);
 
     try {
-      const res = await this.postJson<GithubGraphqlResponse<T>>(
-        'graphql',
-        opts
-      );
+      const res = await this.postJson<GithubGraphqlResponse<T>>(path, opts);
```

Comment:
This change is not really required, but I saw that we have the `path` variable defined and only use it sometimes. This makes use of it always.